src/app/encoder.cpp

```diff
@@ -74,6 +74,59 @@ using namespace chip;
     }                                                                                                                              \
     return result;
 
+#define REPORT_ATTRIBUTE(name, cluster_id, isAnalog, value)                                                                        \
+    BufBound buf = BufBound(buffer, buf_length);                                                                                   \
+    if (_encodeGlobalCommand(buf, destination_endpoint, cluster_id, 0x06))                                                         \
+    {                                                                                                                              \
+        uint8_t direction = 0x00;                                                                                                  \
+        buf.Put(direction);                                                                                                        \
```

Comment:
nit: could also avoid some extra variable usage via:

```
buf.Put<uint8_t>(0x00 /* direction */);
```

or 

```
buf.Put<uint8_t>(0x00); // direction
```

not clear though if this helps or hinders readability.

The usage of hardcoded numbers always bothers me when reviewing this file, but  I assume this is auto-generated or will soon be replaced by auto generated things.